config/elvis.config

```diff
@@ -26,6 +26,11 @@
                     #{regex => "^([a-z][a-z0-9]*_?)*(_SUITE)?$",
                       ignore => []}
                   },
+                  {
+                    elvis_style,
+                    function_naming_convention,
+                    #{regex => "^([a-z]*_?)*(_SUITE)?$"}
```

Comment:
I intentionally removed digits based on "Function names must use only lowercase characters" at https://github.com/inaka/erlang_guidelines#function-names. I also have a rule that forbids digits in function names.

I just want to confirm that you'd like that rule removed, and the regex to match the same one used for module names, which allows digits?

If so, can we update the quotation that I misinterpreted to say "only lowercase characters or digits" ?